vendor/engines/split_accounts/spec/engine_helper.rb

```diff
@@ -0,0 +1,9 @@
+# Rspec configurations for this engine
+RSpec.configure do |config|
+  
+  # Skip split_accounts tests if split_accounts feature is disabled
+  unless SettingsHelper.feature_on?(:split_accounts)
```

Comment:
I'm still a little torn on whether it's redundant to use both the feature flag and gem inclusion to say whether the feature is active. If the gem is loaded, however, I think we should run the tests. And I think we'll need to because circle may still try to run them even if it's not active (we use `vendor/engines/**/spec/**/*_spec.rb` by default in `circle.yml`, but we are restricting that to only one engine in UIC.